evm_arithmetization/src/cpu/kernel/asm/core/exception.asm

```diff
@@ -327,8 +327,16 @@ min_stack_len_for_opcode:
     BYTES 0  // 0x46, CHAINID
     BYTES 0  // 0x47, SELFBALANCE
     BYTES 0  // 0x48, BASEFEE
-    BYTES 1  // 0x49, BLOBHASH
-    BYTES 0  // 0x4a, BLOBBASEFEE 
+    #[cfg(feature = eth_mainnet)]
```

Comment:
It seems weird to me to have `BYTES 1` lying around for an instruction that doesn't exist. Even though as you point this would get caught elsewhere, I think we'd rather not have invalid Kernel bits if we have workarounds